Common/Securities/IMarginCallModel.cs

```diff
@@ -46,6 +57,12 @@ public static class MarginCallModel
 
         private sealed class NullMarginCallModel : IMarginCallModel
         {
+            public SubmitOrderRequest GenerateMarginCallOrder(Security security, decimal netLiquidationValue, decimal totalMargin,
```

Comment:
Will we ever need multiple orders here? Just thinking about generalizing the solution.